ansible/inventory/env/group_vars/all.yml

```diff
@@ -544,3 +544,8 @@ sunbird_desktop_keycloak_client_id: desktop
 sunbird_google_desktop_keycloak_client_id: google-auth-desktop
 sunbird_trampoline_android_keycloak_client_id: trampoline-android
 sunbird_trampoline_desktop_keycloak_client_id: trampoline-desktop
+
+nic_sms_gateway_provider_senderid: "{{core_vault_nic_sms_gateway_provider_senderid}}"
+nic_sms_gateway_provider_username: "{{core_vault_nic_sms_gateway_provider_username}}"
+nic_sms_gateway_provider_password: "{{core_vault_nic_sms_gateway_provider_password}}"
+diksha_dlt_entity_id: "{{core_vault_diksha_dlt_entity_id}}"
```

Comment:
Can we rename this from diksha to sunbird ? or just generic like `dlt_entity_id`